guidelines/backend-coding-standard/folder_structure/serializer/no_bussiness_logic_in_serializer.md

```diff
@@ -1,17 +1,42 @@
 # No bussiness logic in serializer
 # Description
-No bussiness / logic in serializer, should do it in contoller / module level.
+No bussiness logic in serializer, should do it in contoller / module level.
 
 # Example
 ```ruby
-# TODO: adding example here
-
 # 🤔 Avoid this: 
+class SomeUserSerializer
+  attributes :user_status
+
+  def user_status
+    object.student? ? "student" : "not student status"
+  end
```

Comment:
Can we think of something which is more lean to real business logic?
This example is very ambiguous